modules/local/get_chrom_sizes.nf

```diff
@@ -1,20 +1,10 @@
-// Import generic module functions
-include { saveFiles; getProcessName } from './functions'
-
-params.options = [:]
-
 process GET_CHROM_SIZES {
```

Comment:
Can delete and install this module from here https://github.com/nf-core/modules/blob/master/modules/custom/getchromsizes/main.nf

Reply:
Addressed in 318dd3c298e582f87554bee9b33f99a92333a3a4